tests/unit/versioning/Version.spec.js

```diff
@@ -0,0 +1,55 @@
+const { Version } = require('../../../lib/versioning/Version');
+
+describe('Version', () => {
```

Comment:
If you have already reviewed https://github.com/scality/Arsenal/pull/2215, there's no need to do so again as it is a duplicate.